openadapt/app/dashboard/api/recordings.py

```diff
@@ -59,40 +56,37 @@ def recording_detail_route(self) -> None:
         async def get_recording_detail(websocket: WebSocket, recording_id: int) -> None:
             """Get a specific recording and its action events."""
             await websocket.accept()
-            session = crud.get_new_session()
-            with session:
-                recording = crud.get_recording_by_id(recording_id, session)
-
-                await websocket.send_json(
-                    {"type": "recording", "value": recording.asdict()}
-                )
-
-                action_events = get_events(recording, session=session)
-
-                await websocket.send_json(
-                    {"type": "num_events", "value": len(action_events)}
-                )
-
-                for action_event in action_events:
-                    event_dict = row2dict(action_event)
-                    try:
-                        image = display_event(action_event)
-                        width, height = image.size
-                        image = image2utf8(image)
-                    except Exception:
-                        logger.info("Failed to display event")
-                        image = None
-                        width, height = 0, 0
-                    event_dict["screenshot"] = image
-                    event_dict["dimensions"] = {"width": width, "height": height}
-                    if event_dict["key"]:
-                        event_dict["key"] = str(event_dict["key"])
-                    if event_dict["canonical_key"]:
-                        event_dict["canonical_key"] = str(event_dict["canonical_key"])
-                    if event_dict["reducer_names"]:
-                        event_dict["reducer_names"] = list(event_dict["reducer_names"])
-                    await websocket.send_json(
-                        {"type": "action_event", "value": event_dict}
-                    )
-
-                await websocket.close()
+            db = crud.get_new_session(read_only=True)
```

Comment:
As above, what do you think about naming this `session` for consistency?

(This patterns appears multiple times in this PR, I will not address each additional one.)